src/loaders/zarr_utils/types.ts

```diff
@@ -0,0 +1,69 @@
+import SubscribableRequestQueue from "../../utils/SubscribableRequestQueue";
+
+export type TCZYX<T> = [T, T, T, T, T];
+export type SubscriberId = ReturnType<SubscribableRequestQueue["addSubscriber"]>;
+
+export type OMECoordinateTransformation =
+  | {
+      type: "identity";
+    }
+  | {
+      type: "translation";
+      translation: number[];
+    }
+  | {
+      type: "scale";
+      scale: number[];
+    }
+  | {
+      type: "translation" | "scale";
+      path: string;
+    };
+
+export type OMEAxis = {
+  name: string;
+  type?: string;
+  unit?: string;
+};
+
+export type OMEDataset = {
+  path: string;
+  coordinateTransformations?: OMECoordinateTransformation[];
+};
+
+// https://ngff.openmicroscopy.org/latest/#multiscale-md
+export type OMEMultiscale = {
```

Comment:
Great job on making these types.  Kinda surprising that there isn't a thirdparty js lib that captures all these :). Also it's unfortunate that the ome spec is a good deal more general than the sorts of data we hope to present in this viewer. (example - axis names can be just about anything)